### PR TITLE
doc traffic light visualizer

### PR DESCRIPTION
**Pull request by satoshi-ota**
## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Read [pull request guidelines][pull-request-guidelines]
- [x] Code follows [coding guidelines][coding-guidelines]
- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: perception/util/visualizer/traffic_light_visualization/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+## Assumptions / Known limits
+
+## (Optional) Error detection and handling
+
+## (Optional) Performance characterization
+
+## (Optional) References/External links
+
+[1] M. Sandler, A. Howard, M. Zhu, A. Zhmoginov and L. Chen, "MobileNetV2: Inverted Residuals and Linear Bottlenecks," 2018 IEEE/CVF Conference on Computer Vision and Pattern Recognition, Salt Lake City, UT, 2018, pp. 4510-4520, doi: 10.1109/CVPR.2018.00474.
```
Comment (yukkysaito) [inline perception/util/visualizer/traffic_light_visualization/README.md:72]: これは不要かと思われます

Comment (satoshi-ota) [reply perception/util/visualizer/traffic_light_visualization/README.md]: fix in https://github.com/tier4/AutowareArchitectureProposal.iv/pull/628/commits/cce015356d3d6fa307e376a2e98eb3f5c3edd27b

File: perception/util/visualizer/traffic_light_visualization/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+### traffic_light_roi_visualizer
+
+#### Input
+
+| Name                          | Type                                                       | Description              |
+| ----------------------------- | ---------------------------------------------------------- | ------------------------ |
+| `~/input/tl_state`            | `autoware_auto_perception_msgs::msg::TrafficSignalArray`   | status of traffic lights |
+| `~/input/image`               | `sensor_msgs::msg::Image`                                  | input image              |
+| `~/input/rois`                | `autoware_auto_perception_msgs::msg::TrafficLightRoiArray` | input rois               |
+| `~/input/rough/rois` (option) | `autoware_auto_perception_msgs::msg::TrafficLightRoiArray` | input rois               |
```
Comment (taichiH) [inline perception/util/visualizer/traffic_light_visualization/README.md:40]: @satoshi-ota 
It's better to put more information for description. For example, how about the following description?

`~/input/tl_state` : The result of traffic light classification
`~/input/rois` : The ROIs detected by traffic_light_ssd_fine_detector
`~/input/rough/rois` : The ROIs detected by traffic_light_map_based_detector

File: perception/util/visualizer/traffic_light_visualization/README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+None
+
+### traffic_light_roi_visualizer
+
+#### Node Parameters
+
+| Name                    | Type | Default Value | Description                   |
+| ----------------------- | ---- | ------------- | ----------------------------- |
+| `enable_fine_detection` | bool | false         | whether to use fine detection |
```
Comment (taichiH) [inline perception/util/visualizer/traffic_light_visualization/README.md:60]: whether to _visualize_ fine detection result

**Review by taichiH — APPROVED** [review]

File: perception/util/visualizer/traffic_light_visualization/README.md (hunk 3 above)
Comment (satoshi-ota) [reply perception/util/visualizer/traffic_light_visualization/README.md]: fix in https://github.com/tier4/AutowareArchitectureProposal.iv/pull/628/commits/04e51eb14b27d610e0dfa60c3b16e168a0c02f99

File: perception/util/visualizer/traffic_light_visualization/README.md (hunk 2 above)
Comment (satoshi-ota) [reply perception/util/visualizer/traffic_light_visualization/README.md]: fix in https://github.com/tier4/AutowareArchitectureProposal.iv/pull/628/commits/04e51eb14b27d610e0dfa60c3b16e168a0c02f99

**Pull request merged**